common/djangoapps/util/course.py

```diff
@@ -71,16 +71,3 @@ def has_certificates_enabled(course):
     if not settings.FEATURES.get('CERTIFICATES_HTML_VIEW', False):
         return False
     return course.cert_html_view_enabled
-
-
-def should_display_grade(course_overview):
```

Comment:
Note: This code was only used where I remove it. It also checks the same situation that is checked to produce the cert_statuses in the dashboard code but does it in a different way producing unreliable results. https://github.com/edx/edx-platform/blob/48f4f9f4754b17aed25a9b89e870f90d314ebc37/lms/djangoapps/certificates/api.py#L587